src/Main.elm

```diff
@@ -86,9 +96,10 @@ randomWord =
             )
 
 
-try : WordToFind -> UserInput -> Result String Attempt
-try word input =
+try : Language -> WordToFind -> UserInput -> Result String Attempt
+try lang word input =
```

Comment:
There's probably an error string to update as well in this function:

```elm
        Err <| "Sorry, " ++ input ++ " must be a word known from our dictionary"
-- -----------------------------------------------------------------^ French/English
```

Also btw in English language names always take an uppercase letter first.